[#13] Create README.MD file with project description

semyonov-m-y opened the pull request:
Add readme.md, please check

semyonov-m-y committed:
checkstyle_config v.01

semyonov-m-y committed:
checkstyle_config v.01

semyonov-m-y committed:
checkstyle_config v.01

semyonov-m-y committed:
readme.md v.01

semyonov-m-y committed:
[#13] Create README.MD file with project description

zozich commented on readme.md:
```diff
@@ -0,0 +1,27 @@
+The our product is open-sourced software licensed under the MIT license.
+
+## Authors
+* **Andrew Koziulia**
```
Please remove me from authors :)

semyonov-m-y committed:
[#15] Add Travis CI badge to README.MD
[#14] Add Coveralls badge to README.MD

semyonov-m-y committed:
[#15] Add Travis CI badge to README.MD
[#14] Add Coveralls badge to README.MD
[#13] Create README.MD file with project description

semyonov-m-y committed:
readme

semyonov-m-y committed:
readme without badge

Commit:
readme

Commit:
pom
